scene/resources/theme.cpp

```diff
@@ -236,6 +241,24 @@ void Theme::set_default_font_size(int p_font_size) {
 }
 
 // Fallback values for theme item types, configurable per theme.
+void Theme::set_default_theme_base_scale(float p_base_scale) {
```

Comment:
Should there be validation against using say negative scales, or 0?

Reply:
Values at 0.0 and below are not going to be considered valid if the provided API is used (`has_*` methods). We can disallow values below 0.0 here anyway if needed, but 0.0 should be allowed so that the value can be unset from the validity standpoint.

I haven't added any checks because we don't have any checks for font and font size neither, and it works fine for them.